docs/data/sdks/android-kotlin-ampli.md

```diff
@@ -0,0 +1,458 @@
+|`client.instance`| Optional. Specifies an Amplitude instance. By default Ampli creates an instance for you.|
+|`client.apiKey`|Optional. Specifies an API Key. This option overrides the default, which is the API Key configured in your tracking plan.|
+|`client.configuration`|Optional. Specifies the Amplitude configuration. This option overrides the default configuration.|
+
+### Identify
+
+Call `identify()` to identify a user in your app and associate all future events with their identity, or to set their properties.
+
+Just as Ampli creates types for events and their properties, it creates types for user properties.
+
+The `identify()` function accepts an optional `userId`, optional user properties, and optional `options`.
+
+For example your tracking plan contains a user property called `userProp`. The property's type is a string.
+
+=== "Java"
+
+    ```java
+    Ampli.getInstance().identify(userId, Identify.builder()
+      .userProp("A trait associated with this user")
+      .build()
+    );
+    ```
+
+=== "Kotlin"
+
+    ```kotlin
+    ampli.identify(userId, Identify(
+        userProp = "A trait associated with this user"
+    ))
+    ```
+
+The options argument allows you to pass [Amplitude fields](https://developers.amplitude.com/docs/http-api-v2#keys-for-the-event-argument) for this call, such as `deviceId`.
+
+
+=== "Java"
+
+    ```java
+	EventOptions eventOptions =  new EventOptions();
+    eventOptions.setDeviceId("deviceId");
+
+    Ampli.getInstance().identify(
+      userId,
+      Identify.builder().userProp("A trait associated with this user").build(),
+      eventOptions
+    );
+    ```
+
+=== "Kotlin"
+
+    ```kotlin
+	var eventOptions = EventOptions();
+    eventOptions.deviceId = "device-id";
+
+    ampli.identify(
+      userId,
+      Identify(
+          userProp = "A trait associated with this user",
+      ),
+      eventOptions
+    )
+    ```
+
+### Group
+
+--8<-- "includes/editions-growth-enterprise-with-accounts.md"
+
+Call `setGroup()` to associate a user with their group (for example, their department or company). The `setGroup()` function accepts a required `groupType`, and `groupName`.
+
+=== "Java"
+
+    ```java
+    Ampli.getInstance().setGroup("groupType", "groupName");
+    ```
+
+=== "Kotlin"
+
+    ```kotlin
+    ampli.setGroup("groupType", "groupName");
+    ```
+
+--8<-- "includes/groups-intro-paragraph.md"
+
+ Setting a group also sets the 'groupType:groupName' as a user property, and overwrites any existing groupName value set for that user's groupType, and the corresponding user property value. groupType is a string, and groupName can be either a string or an array of strings to indicate that a user is in multiple groups. For example, if Joe is in 'orgId' '10' and '20', then the `groupName` is '[10, 20]').
```

Comment:
this behavior may already changed. setGroup of core SDK now set group_type and group_name to event.groups instead of event.user_proeprties